.github/workflows/release.yml

```diff
@@ -23,8 +57,14 @@ jobs:
           cache: 'maven'
           gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }} # Value of the GPG private key to import
           gpg-passphrase: GPG_PASSPHRASE # env variable for GPG private key passphrase
+      
+      - uses: actions/download-artifact@v3
       - name: Publish to Apache Maven Central
         run: |
+          mv ./natives/* ./cli/
```

Comment:
ah, this is where this is comming from.
Shouldn't we follow the same best-practices like maven and keep artifacts generated during the build only in `target` folders?
The assembly-plugin will also bundle other static files that we need to include into the release. Those will also come from the `target` folder where the `maven-resources-plugin` will have copied them (after potential filtering).

Reply:
Done

Reply:
I do not see this being changed yet. However, it is more a suggestion or nice-to-have improvement.
We can also improve after merge, so I stop blocking this PR with this detail.